packages/mdc-select/mdc-select.scss

```diff
@@ -22,19 +23,26 @@
 @import "@material/ripple/mixins";
 @import "@material/rtl/mixins";
 
-@mixin mdc-select-dd-arrow-svg-bg_($fill-hex-number: 000000, $opacity: .54) {
-  background-image: url(data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%23#{$fill-hex-number}%22%20fill-rule%3D%22evenodd%22%20opacity%3D%22#{$opacity}%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E);
-}
-
 $mdc-select-arrow-padding: 26px;
 $mdc-select-label-padding: 16px;
 $mdc-select-menu-transition: transform 180ms $mdc-animation-standard-curve-timing-function;
 
+.mdc-select {
+  @include mdc-select-dd-arrow-svg-bg_;
+  @include mdc-select-fill-color(rgba(white, .1));
+  @include mdc-select-ink-color(primary);
+  @include mdc-select-label-color(rgba(black, .6));
+  @include mdc-select-bottom-line-color(rgba(black, .5));
+  @include mdc-select-selected-text-color(text-primary-on-light);
+
+  // Focused state colors
+  @include mdc-select-focused-bottom-line-color(primary);
+  @include mdc-select-focused-label-color(primary);
+}
+
 // postcss-bem-linter: define select
 .mdc-select {
```

Comment:
combine this with the above mdc-select...and move the postcss-bem-linter up as well